symfony/framework-bundle/4.2/src/bootstrap.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
this class lives under `src/` which is autoloaded, so composer excepts a class which this file is not. It gives the following error : 
```
In FileLoader.php line 168:
                                                                                                                                                        
  The autoloader expected class "App\bootstrap" to be defined in file "/srv/app/vendor/composer/../../src/bootstrap.php". The file was found but the c  
  lass was not in it, the class name or namespace probably has a typo in /srv/app/config/services.yaml (which is loaded in resource "/srv/app/config/s  
  ervices.yaml").                                                                                                                                       
                                                                                                                                                        

In DebugClassLoader.php line 204:
                                                                                                                                                        
  The autoloader expected class "App\bootstrap" to be defined in file "/srv/app/vendor/composer/../../src/bootstrap.php". The file was found but the c  
  lass was not in it, the class name or namespace probably has a typo.   
```